### ISSUE TITLE
No random seed set in episode 3 and 4?

### ISSUE THREAD

**Issue by ashwinvis** [carpentries-lab/deep-learning-intro#576]
This is used in ep. 2
```py
from numpy.random import seed
seed(1)
keras.utils.set_random_seed(2)
```

but is missing in Episode 3 and 4. Is there a reason?
